src/bootstrap/lib.rs

```diff
@@ -1491,16 +1491,12 @@ impl Build {
         }
         let dst = dstdir.join(src.file_name().unwrap());
         self.verbose_than(1, &format!("Install {:?} to {:?}", src, dst));
-        t!(fs::create_dir_all(dstdir));
-        drop(fs::remove_file(&dst));
         {
             if !src.exists() {
                 panic!("Error: File \"{}\" not found!", src.display());
             }
             let metadata = t!(src.symlink_metadata());
-            if let Err(e) = fs::copy(&src, &dst) {
-                panic!("failed to copy `{}` to `{}`: {}", src.display(), dst.display(), e)
-            }
+            self.copy(src, &dst);
             t!(fs::set_permissions(&dst, metadata.permissions()));
             let atime = FileTime::from_last_access_time(&metadata);
             let mtime = FileTime::from_last_modification_time(&metadata);
```

Comment:
We should update this to avoid updating access times. We should also put a note that the chmod below which updates the perms will also update the original file (not just the "copy" we create) but that this is typically OK for our purposes, since we're using a reasonable permission for both copies in all cases as far as I can tell.